### DISCUSSION TITLE
Extend your Amazon Managed Grafana experience with Grafana community plugins

### DISCUSSION THREAD

**Discussion by VermaPriyanka** [aws/amazon-managed-grafana-roadmap#67] [Announcements]
Amazon Managed Grafana now features a new self-service plugin management experience for Grafana community plugins. Amazon Managed Grafana administrators can now discover and install Grafana community plugins directly from their workspace. Plugins enable you to extend your Grafana experience, unifying data from a wider variety of data sources with visualizations tailored to analyze your unique datasets.

With this release, Amazon Managed Grafana workspace administrators can discover and install plugins from over 300 pre-built Grafana community plugins, to visualize data from their disparate data sources in a single view. You can now glean insights across Amazon services such as Amazon Managed Service for Prometheus, Amazon CloudWatch, right next to third party applications such as Apache Cassandra, PagerDuty or databases such as Redis, Vertica, with visualization panels best suited to represent your data.

The new self-service plugin management experience is supported on all Amazon Managed Grafana workspaces with Grafana version 9.4, across all AWS regions where Amazon Managed Grafana is [generally available](https://docs.aws.amazon.com/grafana/latest/userguide/what-is-Amazon-Managed-Service-Grafana.html#AMG-supported-Regions).

You can enable self-service plugin management for your Amazon Managed Grafana workspace from the [AWS Console](https://console.aws.amazon.com/grafana/home/), SDK, or CLI. Check out the [Amazon Managed Grafana user guide](https://docs.aws.amazon.com/grafana/latest/userguide/grafana-plugins.html) for detailed documentation. To learn more about Amazon Managed Grafana, visit the [product page](https://aws.amazon.com/grafana/) and [pricing page](https://aws.amazon.com/grafana/pricing/).

Want to learn more?
• [What’s New](https://aws.amazon.com/about-aws/whats-new/2023/11/amazon-managed-grafana-community-plugins/)
• [User Guide](https://docs.aws.amazon.com/grafana/latest/userguide/grafana-plugins.html)
• [Blog Post](https://aws.amazon.com/blogs/mt/extend-your-amazon-managed-grafana-experience-with-grafana-community-plugins/)

Feedback
To share any feedback about this feature, please leave a comment in this discussion or submit a new issue.

**Comment by michael-ortiz** [comment]
Any plans to bring on call plugin? I don't see it available on the catalog.
